Sources/NIO/System.swift

```diff
@@ -353,13 +355,27 @@ internal enum Posix {
         }
     }
 
+    @inline(never)
+    public static func ftruncate(descriptor: CInt, size: off_t) throws -> IOResult<CInt> {
+        return try wrapSyscallMayBlock {
```

Comment:
This should only be `wrapSyscall`, as there is no chance of this returning `EAGAIN`.